src/components/checkboxes/index.md.njk

```diff
@@ -65,6 +65,22 @@ You can add hints to checkbox items to provide additional information about the
 
 {{ example({group: "components", item: "checkboxes", example: "hint", html: true, nunjucks: true, open: false, size: "s"}) }}
 
+### Add an option for ‘none’
+
+In some cases, it can be helpful to require users select an checkbox to say that none of the options apply to them, instead of leaving them all unchecked. This also makes sure users do not skip the question by accident.
```

Comment:
```suggestion
In some cases, it can be helpful to require users to select a checkbox to say that none of the options apply to them, instead of leaving them all unchecked. This also makes sure users do not skip the question by accident.
```

Reply:
Good catch, thanks!